scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>As Patrick will tell, we need to sort imports first by type (block for native imports, block for third party imports, block for our utils_cv imports) and then within the blocks alphabetically</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>